python/datasources/kff_vaccination.py

```diff
@@ -55,6 +55,8 @@
 AAPI_STATES = {'Arizona', 'Connecticut', 'District of Columbia', 'Michigan', 'Minnesota', 'Nevada',
                'New Mexico', 'North Carolina', 'Oklahoma', 'South Carolina', 'Virginia'}
 
+KFF_TERRITORIES = ['Guam', 'Puerto Rico', 'Northern Mariana Islands']
```

Comment:
do they not provide data for Samoa, Mariana Islands and Virgin Islands?

Reply:
Nope